python_modules/dagster/dagster_tests/scheduler_tests/test_cron_string_iterator.py

```diff
@@ -75,6 +75,17 @@ def test_cron_iterator_leap_day():
             create_pendulum_time(2023, 11, 1, 1, 45, 0, tz="Europe/Berlin"),
         ],
     ),
+    (
+        "Australia/Lord_Howe",
+        "0 2 * * *",
+        [
+            create_pendulum_time(2023, 9, 29, 2, 0, 0, tz="Australia/Lord_Howe"),
+            create_pendulum_time(2023, 9, 30, 2, 0, 0, tz="Australia/Lord_Howe"),
+            create_pendulum_time(2023, 10, 1, 2, 30, 0, tz="Australia/Lord_Howe"),
+            create_pendulum_time(2023, 10, 2, 2, 0, 0, tz="Australia/Lord_Howe"),
+            create_pendulum_time(2023, 10, 3, 2, 0, 0, tz="Australia/Lord_Howe"),
```

Comment:
the final boss of DST code

Reply:
Yeah it's a pretty wild one. I think there's specific years where countries did special one-off DST changes though. I think that may be the next DST boss.